Plots/MapProjections/NCL_sat_2.py

```diff
@@ -0,0 +1,371 @@
+            z, y = np.where(cond)
+            p = int(round(da.data[z[0]][y[0]]))
+
+            if eType == 'High':
+                lab = plt.text(transformed_locations[x][0], transformed_locations[x][1], "H$_{" + str(p) + "}$", fontsize=fontsize,
+                               horizontalalignment='center', verticalalignment='center')
+            elif eType == 'Low':
+                lab = plt.text(transformed_locations[x][0], transformed_locations[x][1], "L$_{" + str(p) + "}$", fontsize=fontsize,
+                               horizontalalignment='center', verticalalignment='center')
+
+            if horizontal is True:
+                lab.set_rotation('horizontal')
+
+            extremaLabels.append(lab)
+
+        except:
+            continue
+
+    if whitebbox is True:
+        [txt.set_bbox(dict(facecolor='w', edgecolor='none', pad=2)) for txt in extremaLabels]
+
+    return extremaLabels
+
+
+###############################################################################
+
+# Create plot
+
+# Set figure size
+fig = plt.figure(figsize=(8, 8))
+
+# Set global axes with an orthographic projection
+proj = ccrs.Orthographic(central_longitude=270, central_latitude=45)
+ax = plt.axes(projection=proj)
+ax.set_global()
+
+# Add land, coastlines, and ocean features
+ax.add_feature(cfeature.LAND, facecolor='lightgray', zorder=1)
+ax.add_feature(cfeature.COASTLINE, linewidth=.3, zorder=2)
+ax.add_feature(cfeature.OCEAN, facecolor='white')
+ax.add_feature(cfeature.BORDERS, linewidth=.3)
+ax.add_feature(cfeature.LAKES, facecolor='white',
+               edgecolor='k', linewidth=.3)
+
+# Create color map
+colorvalues = [1020, 1036, 1500]
+cmap = colors.ListedColormap(['None', 'lightgray', 'dimgrey'])
+norm = colors.BoundaryNorm(colorvalues, 2)
+
+# Plot contour data
+p = wrap_pressure.plot.contourf(ax=ax,
+                                zorder=2,
+                                transform=ccrs.PlateCarree(),
+                                levels=30,
+                                cmap=cmap, norm=norm,
+                                add_labels=False,
+                                add_colorbar=False)
+
+p = wrap_pressure.plot.contour(ax=ax,
+                               transform=ccrs.PlateCarree(),
+                               linewidths=0.3,
+                               levels=30,
+                               cmap='black',
+                               add_labels=False)
+
+# low pressure contour levels- these will be plotted
+# as a subscript to an 'L' symbol.
+lowClevels = findLocalExtrema(pressure, lowVal=995, eType='Low')
+highClevels = findLocalExtrema(pressure, highVal=1042, eType='High')
+
+# regular pressure contour levels- These values were found by setting
+# 'manual' argument in ax.clabel call to 'True' and then hovering mouse
+# over desired location of countour label to find coordinate
+# (which can be found in bottom left of figure window).
+clevels = [(-145.27, 50.9), (-125.89, 32.33), (-112.62, 19.89),
```

Comment:
Since this plot is so similar to sat_1, and we already have an example of manual label positioning, how about we let MPL pick the label locations automatically for this plot?

Reply:
ok, I'll change it to automatic labeling.